src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -175,6 +176,11 @@ if(DNNL_USE_ACL)
     set(OV_CPU_WITH_ACL ON)
 endif()
 
+if(ENABLE_KLEIDIAI_FOR_CPU)
+    add_definitions(-DOV_CPU_WITH_KLEIDIAI)
+    set(OV_CPU_WITH_KLEIDIAI ON)
```

Comment:
looks like it's not used

Reply:
I plan to use it later

Reply:
ok, why `ENABLE_KLEIDIAI_FOR_CPU` is not enough?